changelog.d/gh-3070.fixed

```diff
@@ -0,0 +1,2 @@
+Autofix: Fixes that span multiple lines will now try to align
+inserted fixed lines with each other.
```

Comment:
But only in fixes applied in OCaml, right? So osemgrep or ast-based autofix?

Reply:
Since https://github.com/semgrep/semgrep/pull/9229, we have been generating all autofixes in OCaml, and then choosing to apply it in either `pysemgrep` or `osemgrep`. So, this should be the source of truth for all autofixes.